examples/llm_and_nlp/unstructured-text.py

```diff
@@ -1,26 +1,39 @@
+# ruff: noqa: E402
 #
-# pip install unstructured[all-docs]
-# libmagic
+# pip install unstructured[all-docs] huggingface_hub[hf_transfer]
 #
 # partition_object supports via unstructured library:
 #
 # "csv", "doc", "docx", "epub", "image", "md", "msg", "odt", "org",
 # "pdf", "ppt", "pptx", "rtf", "rst", "tsv", "xlsx"
 
+import os
+
+os.environ["HF_HUB_ENABLE_HF_TRANSFER"] = "1"
+
+import nltk
+
+nltk.download("punkt_tab")
+nltk.download("averaged_perceptron_tagger_eng")
```

Comment:
[F] These are needed for the "fast" strategy which is the only strategy supported without installing `poppler` or other non-easy-to-install across all platform tools.

Reply:
edit: a new version of `nltk` came out and that meant that `unstructured` could no longer download these files by itself. I have pinned `nltk` back to the working version.